execution.bs

```diff
@@ -4108,39 +4163,235 @@ namespace std::execution {
+    schedule senders obtained from the scheduler, and any operation states
+    obtained from those senders.
+
+11. An asynchronous operation has one or more associated completion schedulers. A
+    <dfn export=true>completion scheduler</dfn> of an asynchronous operation is
+    a scheduler whose associated execution resource is used to execute a
+    completion operation for the asynchronous operation. A value completion
+    scheduler is a scheduler on which an asynchronous operation's value
+    completion operation can execute. Likewise for error completion schedulers
+    and stopped completion schedulers.
+
+12. A sender has an associated queryable object known as its <dfn
+    export=true>attributes</dfn> that describe various characteristics of the
+    sender and of the asynchronous operation(s) it produces. A sender can
+    expose the completion scheduler(s) of its asynchronous operations via
+    its attributes. If present, a value completion scheduler attribute
+    indicates that the sender's asynchronous operations only execute value
+    completion operations on that scheduler's associated execution resource.
+    Likewise for error and stopped completion scheduler attributes.
+    A schedule sender is required to have a value completion scheduler attribute
+    whose value is equal to the scheduler that produced the schedule sender.
+
+13. A <dfn export=true>completion signature</dfn> is a function type that
+    describes a completion operation. An asychronous operation has a finite set
+    of possible completion signatures. The completion signature's return type is
+    the completion tag associated with the completion function that executes the
+    completion operation. The completion signature's argument types are the
+    types and value categories of the asynchronous operation's result datums,
+    determined as if by `decltype`. Together, a sender type and an environment
+    type `E` uniquely determine the set of completion signatures of the
+    operation state that results from connecting the sender with a receiver
+    whose environment has type `E`. <span class="wg21note">The type of the
+    receiver does not effect an operation state's completion signatures, only
+    the type of the receiver's environment.</span>
+
+14. A <dfn export=true>sender algorithm</dfn> is a function that takes and/or
+    returns a sender. There are three categories of sender algorithms:
+
+    * A <dfn export=true>sender factory</dfn> is a function that takes
+        non-senders as arguments and that returns a sender.
+
+    * A <dfn export=true>sender adaptor</dfn> is a function that takes one or
+        more senders and a (possibly empty) set of additional arguments, and
+        that returns a sender. The returned sender is the <dfn
+        export=true>parent sender</dfn> and the sender arguments to the sender
+        adaptor are its <dfn export=true lt="child sender">child senders</dfn>.
```

Comment:
> The returned sender is the parent sender and the sender arguments to the sender adaptor are its child senders.

Are there sender adaptors for which the argument senders are not necessarily children?
Or would such algorithms just not be classed as sender adaptors?